Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM sub2u/ruby-2.3.1-node-6.8.0
+MAINTAINER sub2u
+
+COPY ./ app/
```

Comment:
Wouldn't it be more convenient to mount directory instead of coping over all the things inside container?